docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -545,35 +548,33 @@ one level of *Fatigue*. Reshuffle your deck and draw all Checks with the
 Lower Hand. Each time you need to reshuffle your deck in this way adds
 another level of Fatigue. If you were to suffer a fourth level of Fatigue,
 you are Knocked Out instead. This signifies your character running out of
-Stamina during a long day. 
+Stamina during a long day.
 
-### Discard Mechanics
+### Discard Powers
 
-Each Dealer may have an additional Burn Mechanic. Dealers may discard cards as
+Each Dealer may have one Discard Power. Dealers may discard cards as
 a Minor Action for additional benefits in combat starting at Level 2. For
 every 2 cards discarded, up to a limit of 4, an Upper/Lower Hand +1 is
 granted described below. These effects do not stack; only one may be applied
-to a character at a time.
+to a character at a time. Cards are discarded without being turned over, and cannot
+yeild Fate Cards.
 
-Below is a list of possible Discard Mechanics, determined by Role. Work with your GM if you
-would like to use a different Burn Mechanic instead.  
+Below is a list of possible Discard Powers, determined by Role. Work with your GM if you
+would like to use a different Discard Power instead.  
 <!-- Specifying in combat until we can revsit non-combat uses of agro and thwart -->
 
-- Clubs ♣️, Aggro: Defense Dealers have a strong foundation and can take the big hits.
-  They can choose to *bury* N cards to force all creatures within 2 spaces to make
-  their next Check to attack targets besides the Defense Dealer at Lower Hand (N/2).
+- Aggro: Discard N cards to force all creatures within 2 spaces to make
+  their next Check to attack targets besides the triggering Dealer at Lower Hand (N/2).
 
-- Diamonds ♦️, Intensify: Casters Dealers have explosive damage but they run out of gas
-  really fast. They can choose to *burn* N cards to make their next Check at Upper
-  Hand (N/2).
+- Help: Discard N cards to grant an ally within 6 spaces Upper Hand (N/2) on all Checks
+  for their next turn.
```

Comment:
Should this be exclusively on the characters turn or is it an effect that lasts until the end of their next turn (to allow a Contested Check or Save before their turn to have Upper Hand as well?)